src/work-allocation-2/containers/task-assignment-confirm/task-assignment-confirm.component.html

```diff
@@ -0,0 +1,42 @@
+<main class="govuk-main-wrapper" id="main-content" role="main">
+  <h1 class="govuk-heading-xl">{{verb}} task</h1>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+
+      <h2 class="govuk-heading-m">Check your changes</h2>
+      <p>Check you are {{verb.toLowerCase()}}ing the work to the right person</p>
+
+      <table class="govuk-table">
+        <thead class="govuk-table__head">
+          <th class="govuk-table__header">Case name</th>
+          <th class="govuk-table__header">Case category</th>
+          <th class="govuk-table__header">Location</th>
+          <th class="govuk-table__header">Task</th>
+          <th class="govuk-table__header">Date</th>
+          <th class="govuk-table__header">Person</th>
+          <th class="govuk-table__header"></th>
+        </thead>
+
+        <tbody class="govuk-table__body">
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell">Joe Blogs</td>
+            <td class="govuk-table__cell">Protection</td>
+            <td class="govuk-table__cell">Taylor House</td>
+            <td class="govuk-table__cell">Review respondent interview</td>
+            <td class="govuk-table__cell">
+              <strong class="govuk-tag govuk-tag--red">
+                01/01/1971
+              </strong>
+            </td>
+            <td class="govuk-table__cell">Judge Birch</td>
```

Comment:
hardcoded?

Reply:
Yeah agree with @rayliangatos  can you use mock data here?

Reply:
I spoke on stand-up about this and it was agreed that I should do this. I can change it but wish people would make up their mind.

Reply:
After discussion today on stand-up it was agreed to do this and that a mock was needed at this stage.

Reply:
Why does this need a mock? Isn't this just showing the user the very selection they made on the previous screen? Why is the data not simply being passed across by some straightforward means?

A mock makes no sense to me at all. Where does the need to call an API feature into this?

Reply:
Moreover, you're making a _selection_ on the previous page, which suggests you're going to send that selection to an endpoint somewhere so it can subsequently be retrieved from that endpoint (i.e., the mock) but I can't imagine the WA API is going to offer an endpoint for a temporary selection, which means you'd have to implement the temporary store within MC instead... so what you _really_ want is that temporary store to be _implemented_, not mocked.

Architecturally, that seems like a strange thing to do when you can simply pass the details from one page to the next... or even have two modes on a single page ("select" and "confirm"). 🤦‍♂️

Reply:
Actually, how about this for a mock in the .ts file:
```
public mockTask: Task = {
  id: 'bob',
  case_id: '1234567890123456',
  caseName: 'Joe Bloggs',
  caseCategory: 'Protection',
  location: 'Taylor House',
  taskName: 'Review respondent interview',
  dueDate: new Date('1971-01-01T00:00:00'),
  actions: []
};

public mockCaseworker: Caseworker = {
  idamId: 'fred',
  firstName: 'Judge',
  lastName: 'Birch',
  email: 'judge.birch@hmcts.gov.uk',
  location: { id: 'TH', locationName: 'Taylor House', services: [] };
};
```

And then do the bindings in the template. Job done.

Reply:
Discussed in the call today that it was agreed hard coding is fine on Friday's call [Which I was not aware of]